src/styles/pagination.m.css

```diff
@@ -0,0 +1,34 @@
+@import './shared/variables.css';
+
+.navigation {
```

Comment:
I changed it to use flex - we'll have to revisit it when we're actually rendering the pagination widget in the grid and can see the results.